pkg/evaluator/evaluator_test.go

```diff
@@ -690,6 +690,27 @@ end
 	}
 }
 
+func TestHas(t *testing.T) {
```

Comment:
I think you need some error tests too for wrong types if you are going to do the type checking in the evaluator.

Reply:
ok, done.